docs/modules/ROOT/pages/getting-started.adoc

```diff
@@ -52,6 +53,7 @@ kotlin {
 <1> xref:core:source-code.adoc[Displaying & Animating Source Code]
 <2> xref:plugins:speaker-window.adoc[Plugin: Speaker Window]
 <3> xref:plugins:laser.adoc[Plugin: Laser]
+<3> xref:core:slide.adoc#bullet-points[Material widgets]
```

Comment:
Should be **<4>**
Should refer to the Wiget page at large (not bullet-points) as we may add other widgets later.